config.go

```diff
@@ -128,178 +89,72 @@ func (config *TGFConfig) InitAWS(profile string) error {
 }
 
 // SetDefaultValues sets the uninitialized values from the config files and the parameter store
+// Priorities (Higher overwrites lower values):
+// 1. SSM Parameter Config
+// 2. Secrets Manager Config (If exists, will not check SSM)
+// 3. .tgf.user.config
```

Comment:
It is not hidden, its tgf.user.config

I should override .tgf.config

Reply:
From what I understand, you reversed the way of evaluating precedence. We should also reverse the evaluation of config files.

Reply:
I reversed it but the logic is different. 
You were setting values only if they were unset: That way you set explicit variables and they are not set by other methods
What I do is Unmarshal every layer on top of the others, that way later calls will overwrite values previously set.